### PR TITLE
Makefile updates

### PR DESCRIPTION
**Pull request by krishvoor**
This PR will auto-update the based on the ARCH and pass them along 
to Makefile target.

### CODE REVIEW

**Comment by svetsa-rh** [comment]
@krishvoor Changes look fine to me, but DPTP @openshift/test-platform-maintainers  needs to review your changes and approve. 
cc: @jupierce @jmguzik

**Comment by svetsa-rh** [comment]
@krishvoor If needed, you can also reach out to the team on `#forum-testplatform` in slack for PR review.

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 1]
```diff
@@ -471,7 +471,6 @@ presubmits:
       ci.openshift.io/generator: cluster-init
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-release-master-build04-dry
-    optional: true
```
Comment (smg247) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:474]: Please revert this. The job has been intentionally made optional due to the cluster outage. The `build-clusters` job is also optional, you can ignore that it is failing.

Comment (krishvoor) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:474]: Thanks for confirming, updating this now.

**Pull request merged**